Src/WitsmlExplorer.Api/Jobs/DownloadAllLogDataJob.cs

```diff
@@ -1,11 +1,12 @@
+using System;
 using System.Collections.Generic;
 
 using WitsmlExplorer.Api.Models;
 
 namespace WitsmlExplorer.Api.Jobs;
 
 /// <summary>
-/// Job for downloading all log data.  
+/// Job for downloading all log data.
 /// </summary>
 public record DownloadAllLogDataJob : Job
```

Comment:
Now that it's not just for the whole log, can you remove _All_ to rename all occurrences of _DownloadAllLogData_ to _DownloadLogData_?

Reply:
fixed